dbcon/mysql/ha_mcs_pushdown.cpp

```diff
@@ -407,7 +396,7 @@ create_columnstore_group_by_handler(THD* thd, Query* query)
     // MCOL-2178 Disable SP support in the group_by_handler for now
     // Check the session variable value to enable/disable use of
     // group_by_handler. There is no GBH if SH works for the query.
-    if (get_select_handler(thd) || !get_group_by_handler(thd) || (thd->lex)->sphead)
+    if (((int) get_select_handler(thd) == 1) || !get_group_by_handler(thd) || (thd->lex)->sphead)
```

Comment:
Could you replace 1 with the enum value name? It will make the text more readable.

Reply:
I agree. Done.